AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/data/main/FooterType.java

```diff
@@ -0,0 +1,32 @@
+package com.applovin.enterprise.apps.demoapp.data.main;
+
+import android.os.Build;
+
+import com.applovin.enterprise.apps.demoapp.BuildConfig;
+import com.applovin.sdk.AppLovinSdk;
+
+public class FooterType
+        implements ListItem
+{
+    /**
+     * @return The device details: App Version, SDK Version, OS Version
+     */
+    public String getAppDetails()
+    {
+        String appVersion = BuildConfig.VERSION_NAME;
+        String sdkVersion = AppLovinSdk.VERSION;
+        String versionName = Build.VERSION_CODES.class.getFields()[android.os.Build.VERSION.SDK_INT].getName();
```

Comment:
nit: This feels a bit [unsafe](https://stackoverflow.com/a/55946200). May want to try-catch this or something.

Reply:
updated in java/kotlin files

Reply:
In the SO reply mentioned above, it seems like based on the API level, this may or may not work and have a different solution. Do you mind checking if this would work on older versions of Android.

Reply:
ah sorry I didn't realize that was a link.

Reply:
Now properly updated